app/cdash/config/config.php

```diff
@@ -0,0 +1,339 @@
+<?php
+/*=========================================================================
+  Program:   CDash - Cross-Platform Dashboard System
+  Module:    $Id$
+  Language:  PHP
+  Date:      $Date$
+  Version:   $Revision$
+
+  Copyright (c) Kitware, Inc. All rights reserved.
+  See LICENSE or http://www.cdash.org/licensing/ for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE. See the above copyright notices for more information.
+=========================================================================*/
+
+/** WARNING: It's recommended to create a config.local.php file and leave
+ * this file as is.
+ * If creating the config.local.php from config.php make sure you DELETE
+ * any text after the 'DO NOT EDIT AFTER THIS LINE' otherwise your
+ * configuration file will be referencing each other. */
+
+// This file is 'config.php', in the directory 'config', in the root.
+// Therefore, the root of the CDash source tree on the web server is:
+@include_once dirname(__DIR__) . '/vendor/autoload.php';
+include_once dirname(__FILE__) . '/../bootstrap/cdash_autoload.php';
+
+$CDASH_ROOT_DIR = str_replace('\\', '/', dirname(dirname(__FILE__)));
+set_include_path(get_include_path() . PATH_SEPARATOR . $CDASH_ROOT_DIR);
+
+
```

Comment:
We used to call `date_default_timezone_set('UTC')` here because PHP would complain at us otherwise. Is this something that Laravel handles for us instead?

Reply:
https://github.com/Kitware/CDash/blob/ffa52147dac439478f7b3c25fb3245e52e0b91f0/config/app.php#L58-L70

Reply:
👍